x/authz/client/cli/tx.go

```diff
@@ -174,6 +191,7 @@ Examples:
 	cmd.Flags().StringSlice(FlagAllowedValidators, []string{}, "Allowed validators addresses separated by ,")
 	cmd.Flags().StringSlice(FlagDenyValidators, []string{}, "Deny validators addresses separated by ,")
 	cmd.Flags().Int64(FlagExpiration, time.Now().AddDate(1, 0, 0).Unix(), "The Unix timestamp. Default is one year.")
+	cmd.Flags().String(FlagAllowedAuthorizations, "", "Allowed authorizations for Count Authorization")
```

Comment:
This should be a numerical type. Above, you're getting it using `GetInt32`, and the message field is `int32`:
```suggestion
	cmd.Flags().Int32(FlagAllowedAuthorizations, 0, "Allowed authorizations for a Count Authorization")
```